### PR TITLE
feat: add custom install command

### LINKED ISSUE #236: Allow custom install command
**Not a bug, but a feature request!**

## What?

Allow me to use a custom install command, such as:
`yarn install --non-interactive --frozen-lockfile --production=true --cache-folder ~/.cache/yarn`

## Why?

The default command for both yarn and npm are hard-coded with a sensible default, but I'd like to be able to break out of that beyond just choosing yarn or npm. In this example, I've defined a yarn cache location that differs from the default preventing the Orb from using my dependency cache :(

### PR DESCRIPTION
**Pull request by bahmutov**
- closes #236 

```yml
version: 2.1
orbs:
  cypress: cypress-io/cypress@1
workflows:
  build:
    jobs:
      - cypress/run:
          install-command: 'yarn install --frozen-lockfile --production=true'
```

### CODE REVIEW

**Comment by bahmutov** [comment]
What do you think @kylemh

File: src/orb.yml [hunk 1]
```diff
@@ -607,6 +644,19 @@ examples:
...
+      Install dependencies using your own custom command.
+    usage:
+      version: 2.1
+      orbs:
+        cypress: cypress-io/cypress@1
+      workflows:
+        build:
+          jobs:
+            - cypress/run:
+                install-command: 'yarn install --frozen-lockfile --production=true'
```
Comment (kylemh) [inline src/orb.yml]: I would do `production=false` since Cypress is almost always a devDep

File: src/orb.yml [hunk 2]
```diff
@@ -1091,8 +1141,5 @@ examples:
 # workflows:
 #   build:
 #     jobs:
-#       - install
 #       - run:
-#           requires:
-#             - install
-#           attach-workspace: true
+#           install-command: 'yarn install --non-interactive --frozen-lockfile --production=true'
```
Comment (kylemh) [inline src/orb.yml]: same here

**Comment by kylemh** [comment]
Aw shoot. I was gonna do it for ya! Hell yeah, though - thank you!

My last note would be... are you [trying to normalize](https://github.com/cypress-io/circleci-orb/issues/148) on `-` or `_`

Curious to know when you're ready for a breaking change PR? I'd love to make a cut for ya!

**Comment by bahmutov** [comment]
For normalize - I would prefer going to `-` to me it looks better when reading, but it would be a breaking change

**Pull request merged**